packages/language-server/src/plugins/svelte/features/getCompletions.ts

```diff
@@ -28,13 +41,40 @@ export function getCompletions(
     const notPreceededByOpeningBracket = !/[\s\S]*{\s*[#:/@]\w*$/.test(
         lastCharactersBeforePosition,
     );
-    if (isInStyleOrScript || notPreceededByOpeningBracket) {
+    if (isInStyleOrScript) {
         return null;
     }
 
-    const triggerCharacter = getTriggerCharacter(lastCharactersBeforePosition);
-    // return all, filtering with regards to user input will be done client side
-    return getCompletionsWithRegardToTriggerCharacter(triggerCharacter, svelteDoc, offset);
+    if (notPreceededByOpeningBracket) {
+        return getComponentDocumentationCompletions();
+    }
+
+    return getTagCompletionsWithinMoustache();
+
+    /**
+     * Get completions for special svelte tags within moustache tags.
+     */
+    function getTagCompletionsWithinMoustache() {
+        const triggerCharacter = getTriggerCharacter(lastCharactersBeforePosition);
+        // return all, filtering with regards to user input will be done client side
+        return getCompletionsWithRegardToTriggerCharacter(triggerCharacter, svelteDoc, offset);
+    }
+
+    function getComponentDocumentationCompletions() {
+        if (!lastCharactersBeforePosition.includes(HTML_COMMENT_START)) {
+            return null;
+        }
+
+        const commentStartIndex = lastCharactersBeforePosition.lastIndexOf(HTML_COMMENT_START);
+        const text = lastCharactersBeforePosition.substring(
+            commentStartIndex + HTML_COMMENT_START.length
+        ).trimLeft();
+
+        if (componentDocumentationCompletion.label.includes(text)) {
+            return CompletionList.create([componentDocumentationCompletion], true);
```

Comment:
Is there a reason why you set the completion to `incomplete`? Are there cases where other completions would no longer show up if we set it to `false`?

Reply:
You're right, don't need that. I got what I want to achieve with `filterText` and didn't find out I don't need `isIncomplete` anymore